app/controllers/content_items_controller.rb

```diff
@@ -24,6 +24,35 @@ class ContentItemsController < ApplicationController
   def show
     load_content_item
 
+    # TEMPORARY (author: richard.towers, expected end date: November 30 2023)
+    # Content specific AB test for the Find your UTR number page
+    if @content_item.base_path == "/find-utr-number"
+      ab_test = GovukAbTesting::AbTest.new(
+        "find_utr_number_video_links",
+        dimension: 300, # TODO: which dimension should we use?
+        allowed_variants: %w[HelpText VideoLink],
+        control_variant: "HelpText",
+      )
+      @requested_variant = ab_test.requested_variant(request.headers)
+      @requested_variant.configure_response(response)
+
+      if @requested_variant.variant? "VideoLink"
+        # NOTE: this is a fragile way of doing an AB test on content.
+        #
+        # Any change to the base content, or to the way the content is rendered
+        # could potentially break the B variant of the test, and result in both
+        # variants being the same.
+        # We're aware of this risk, and we're going to be careful in this one off
+        # situation. This is not a sustainable way of doing AB tests in the
+        # future.
+        @content_item.body.sub!(
+          /<li>\s*in\ the\s+<a\ href="[^"]*"><abbr\ title="[^"]+">HMRC<\/abbr>\s+app<\/a>\s*<\/li>/,
+          '<li>in the <a href="https://www.gov.uk/guidance/download-the-hmrc-app"><abbr title="HM Revenue and Customs">HMRC</abbr> app</a> - watch a <a href="https://www.youtube.com/watch?v=LXw9ily9rTo">video about finding your UTR number in the app</a>',
```

Comment:
Forgot the closing `</li>` tag here (although technically they're optional in HTML now 😅)